composer.json

```diff
@@ -16,6 +16,12 @@
     "autoload-dev": {
         "classmap": [ "app/tests/WebTestCase.php" ]
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:ElectricMaxxx/sonata-admin-integration-bundle.git"
+        }
+    ],
```

Comment:
I need to move it today until that it lives in my space :-)
Will move when we have the name.